src/Symfony/Component/Finder/Finder.php

```diff
@@ -373,6 +374,17 @@ public function ignoreVCS($ignoreVCS)
         return $this;
     }
 
+    public function ignoreVCSIgnored($ignoreVCSIgnored)
```

Comment:
This method will be part of the public API of the class. Please add a brief PHPDoc block documenting what the method does. Also, please add a `@return $this` annotation, so IDEs know that this method implements a fluent interface.